modules/ROOT/pages/_partials/archive-api.adoc

```diff
@@ -0,0 +1,9 @@
+You can access API analytics metrics data directly via the Anypoint Monitoring Archive API. Using this API, you can navigate and discover your API metrics data using a directory hierarchy.
```

Comment:
this include (archive-api.adoc) is included in 
* modules/ROOT/pages/am-apis.adoc
* modules/ROOT/pages/api-analytics-dashboard.adoc